### DISCUSSION TITLE
Overlay icons on tortoisegit

### DISCUSSION THREAD

**Discussion by francamorais** [StefSuperGit/Tafe230212#2] [General]
Hi Stefan,
My overlay icons are not working for tortoise git. Do you know what could be happening?

**Comment by StefSuperGit** [comment]
I've had a similar issue before. I sort the advice from this link:
https://stackoverflow.com/questions/25156238/tortoisegit-not-showing-icon-overlays

You need to change some things in the registry editor which can be a bit daunting. If you have any issues I will be happy to have a look.

Regards,

Stefan
